qdrant-landing/content/blog/qdrant-1.10.x.md

```diff
@@ -0,0 +1,247 @@
+        "using": "full"
+        "limit": 100,
+    },
+    "query": [           // <─┐
+        [0.1, 0.2, ...], // < │
+        [0.2, 0.1, ...], // < ├─ multi-vector
+        [0.8, 0.9, ...]  // < │
+    ],                   // <─┘       
+    "using": "colbert",
+    "limit": 10
+}
+```
+
+**Note:** *The multivector feature is not only useful for ColBERT; it can also be used in other ways.*</br> 
+For instance, in e-commerce, you can use multivector to store multiple images of the same item. This serves as an alternative to the [group-by](/documentation/concepts/search/#grouping-api) method. 
+
+## Sparse Vectors Compression
+
+In version 1.9, we introduced the `uint8` [vector datatype](/documentation/concepts/collections/#vector-datatypes) for sparse vectors, in order to support pre-quantized embeddings from companies like JinaAI and Cohere. 
+This time, we are introducing a new datatype **for both sparse and dense vectors**, as well as a different way of **storing** these  vectors. 
+
+**Datatype:** Sparse and dense vectors were previously represented in larger `float32` values, but now they can be turned to the `float16`. `float16` vectors have a lower precision compared to `float32`, which means that there is less numerical accuracy in the vector values - but this is negligible for practical use cases. 
+
+These vectors will use at least half the memory of regular vectors, which can significantly reduce the footprint of large vector datasets. Operations can be faster due to reduced memory bandwidth requirements and better cache utilization. This can lead to faster vector search operations, especially in memory-bound scenarios.
+
+When creating a collection, you need to specify the `datatype` upfront:
+```http
+PUT /collections/{collection_name}
+{
+    "vectors": {
+      "size": 1024,
+      "distance": "Cosine",
+      "datatype": "float16"
+    }
+}
+```
+**Storage:** On the backend, we implemented bit packing to minimize the bits needed to store data, crucial for handling sparse vectors in applications like machine learning and data compression. For sparse vectors with mostly zeros, this focuses on storing only the indices and values of non-zero elements. 
+
+You will benefit from a more compact storage and higher processing efficiency. This can also lead to reduced dataset sizes for faster processing and lower storage costs in data compression.
+
+## New Rust Client 
+Qdrant’s reshaped Rust client is now more accessible and easier to use. We have focused on putting together a minimalistic user experience. Its ownership model ensures memory safety without needing a garbage collector, making it ideal for managing intensive computations and large datasets typical of vector databases. Additionally, Rust supports safe concurrent execution, which is crucial for handling multiple simultaneous requests efficiently.
+
+<p align="center">
+    <a href="https://github.com/qdrant/rust-client">Rust Client Repo</a> and 
```

Comment:
@timvisee Will the links for the docsite be the same?

Reply:
Yes!

- https://docs.rs/qdrant-client

I think it would be nicer to render the `docs.rs` link before the repository, because the new documentation is more interesting.